controllers/check_auth.lua

```diff
@@ -0,0 +1,21 @@
+local Users  = require "models.users"
+
+return function(self)
+  
+  -- Verify Authorization
+	if self.session.name then
+		local user = Users:get_user(self.session.name)
+
+		if user then
+			user.password = nil
```

Comment:
What does this line do?

Reply:
removes the plaintext password from memory

Reply:
Sorry, here it removes the hashed password from memory. Same goal though, keep the password in memory as short a time as possible.